src/Microsoft.VisualStudio.Web.CodeGeneration.Design/Microsoft.VisualStudio.Web.CodeGeneration.Design.csproj

```diff
@@ -9,9 +9,12 @@
     <OutputType>exe</OutputType>
     <PackageTags>aspnetcore;codegenerator;scaffolding;visualstudioweb</PackageTags>
     <PackageId>Microsoft.VisualStudio.Web.CodeGeneration.Design</PackageId>
-    <RuntimeIdentifier Condition=" '$(TargetFramework)' != 'netcoreapp2.0' ">win7-x86</RuntimeIdentifier>
   </PropertyGroup>
 
+  <PropertyGroup Condition=" '$(TargetFramework)' != 'netcoreapp2.0' ">
```

Comment:
Turns out this doesn't work. It solves the problem of MVC loading the assembly, but scaffolding itself fails when it tries to run the user code.